pkg/sql/sem/tree/show.go

```diff
@@ -137,6 +137,15 @@ func (node *ShowDatabases) Format(ctx *FmtCtx) {
 	}
 }
 
+// ShowEnums represents a SHOW ENUMS statement.
+type ShowEnums struct {
```

Comment:
nit just put this on on line `type ShowEnums struct {}`

Reply:
Done